apps/client/src/app/pages/inventory-optimizer/inventory-optimizer/inventory-optimizer.component.ts

```diff
@@ -120,14 +120,41 @@ export class InventoryOptimizerComponent {
   }
 
   constructor(private inventoryFacade: InventoryFacade,
-              @Inject(INVENTORY_OPTIMIZER) private optimizers: InventoryOptimizer[],
-              private lazyData: LazyDataService, private message: NzMessageService, private translate: TranslateService) {
+    @Inject(INVENTORY_OPTIMIZER) private optimizers: InventoryOptimizer[],
+    private lazyData: LazyDataService, private message: NzMessageService, private translate: TranslateService) {
   }
 
   private getContainerName(item: InventoryItem): string {
     return item.retainerName || this.inventoryFacade.getContainerName(item.containerId);
   }
 
+  public getExpansions(): string[] {
+    let expansions: any[] = this.lazyData.patches.map(p => {
```

Comment:
```suggestion
    const expansions: any[] = this.lazyData.patches.map(p => {
```

Never reassigned, prefer `const` over `let` (the linter won't accept it in CI).